test/db/cmd/cmd_list

```diff
@@ -1,40 +1,539 @@
+EOF
+RUN
+
+NAME=Print the lang plugins
+FILE==
+CMDS=Ll
+EXPECT=<<EOF
+c: C language extension (LGPL)
+cpipe: rzpipe scripting in C (LGPL)
+vala: Vala language extension (LGPL)
+rust: Rust language extension (MIT)
+zig: Zig language extension (MIT)
+spp: SPP template programs (MIT)
+pipe: Use #!pipe node script.js (LGPL)
+lib: Load libs directly into rizin (LGPL)
+EOF
+RUN
+
+NAME=Print the lang plugins in JSON
```

Comment:
please also test the bin plugins in JSON, because that part seems the most "complicated"